retext.1

```diff
@@ -0,0 +1,20 @@
+.TH RETEXT 1
```

Comment:
1. This should be upstreamed
2. This doesn't belong in the flatpak anyway

Reply:
This is a source file, used for update translation.

Reply:
To be clear, `flatpak` doesn't expose man files, just remove it as no user will ever see it.